src/api/v1/prerequisites/is_child_user.ts

```diff
@@ -0,0 +1,56 @@
+/*
+ * "user_details-child" route pre-requisite
+ *
+ * Determines if the authenticated user is trying to access an user
+ * which is actually a "child" entity.
+ *
+ * Conditions under which this is true for user X attemptying to access:
```

Comment:
Is this prereq intended for accessing unserialised user data? If so I think we should mention that FUNDING_BODY_ADMIN's are unauthenticated for this reason.

If not then we should add a note to add their use case at a later date

Reply:
What do you mean by "accessing unserialised user data"

Reply:
User data that allows the user to be identified. Due to us not collecting gdpr consent for  funding bodies they cannot access personal info data about users.

Reply:
OK. But they will be allowed to view aggregate or anonymised data. We may need to add new permissions for that. Maybe `user_aggregates-child:read` or similar. I'll add a note to the description here.

Also, for reference: [serialisation](https://en.wikipedia.org/wiki/Serialization)